release-notes/opensearch-neural-search.release-notes-2.19.0.0.md

```diff
@@ -0,0 +1,30 @@
+## Version 2.19.0.0 Release Notes
+
+Compatible with OpenSearch 2.19.0
+
+### Features
+* Pagination in Hybrid query ([#1048](https://github.com/opensearch-project/neural-search/pull/1048))
+* Implement Reciprocal Rank Fusion score normalization/combination technique in hybrid query ([#874](https://github.com/opensearch-project/neural-search/pull/874))
+
```

Comment:
there is no empty line between the sections of the release notes, can you please remove it

Reply:
Sure.